_includes/navMenus/learn.html

```diff
@@ -0,0 +1,90 @@
+<ul class="menuGroup learn">
+    <ul class="wrapperSection">
+        <div class="navItemGroupWrapper">
+            <div class="title">Pick your learning path</div>
+            <span class="subline">A wide range of resources to get you started</span>
+            <a href="https://developer.confluent.io/learn-kafka/" class="navItemGroupCta">
+                Start Learning
+                <img src="/tutorials/assets/img/icon-arrow-right-white.svg" width="15" height="15"/>
+            </a>
+        </div>
+    </ul>
+    <ul class="navItemMainColumns">
+        <ul>
```

Comment:
not sure you can nest a `ul` within a `ul` directly

Reply:
The book says not to use `ul` under `ul` directly, but works anyway ...?

Reply:
Updated.